tools/prepare-source.sh

```diff
@@ -18,7 +18,7 @@ test -e "$TOOLS_PATH/goimports" || GOBIN=$TOOLS_PATH $GO_BINARY install golang.o
 GOBIN=$TOOLS_PATH $GO_BINARY generate -x -mod=mod ./...
 
 # ... the code is formatted correctly, ...
-goimports -w internal cmd
+"$TOOLS_PATH/goimports" -w internal cmd
```

Comment:
Please drop this change.

I just realized there is a problem with the script when goimports is missing. Let me fix that in a different patch.